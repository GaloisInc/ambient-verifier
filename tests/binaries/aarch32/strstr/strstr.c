#include <string.h>
#include "ambient_assert.h"

int main() {
  char *haystack = "ABC123";
  char *needle = "C12";
  char *res = strstr(haystack, needle);
  ambient_assert(res[0] == 'C' &&
                 res[1] == '1' &&
                 res[2] == '2' &&
                 res[3] == '3' &&
                 res[4] == '\0');
  return 0;
}
