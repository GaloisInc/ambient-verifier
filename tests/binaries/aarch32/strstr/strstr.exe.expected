successful: 0
failed: 0
overrideDir: tests/binaries/aarch32/strstr/overrides
